docker/tidb-sysbench/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM alpine:3.9
+MAINTAINER PingCAP <cloud@pingcap.com>
+
+ENV sysbench_version 1.0.17
+
+# Install sysbench
+# gcc and mariadb-dev install shared libraries used at runtime
+RUN apk add gcc mariadb-dev
```

Comment:
It's better to install all the packages in a single `RUN` instruction and with `--no-cache` argument

Reply:
done now